steampy/client.py

```diff
@@ -311,6 +313,17 @@ def fetch_price(self, item_hash_name: str, game: GameOptions, currency: str = Cu
             raise TooManyRequests("You can fetch maximum 20 prices in 60s period")
         return response.json()
 
+    @login_required
+    def get_my_market_listings(self) -> dict:
+        response = self._session.get("%s/market" % SteamClient.COMMUNITY_URL)
+        if response.status_code == 200:
+            assets_descriptions = json.loads(text_between(response.text, "var g_rgAssets = ", ";"))
+            listing_id_to_assets_address = get_listing_id_to_assets_address_from_html(response.text)
```

Comment:
We are getting a dict "listing_id" -> ["appid", "contextid", "assetid"]
Example [here](https://gist.github.com/Ermans/f07731e7e034ae7b7455be01ce186c59)